CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+
+__Improving comments__ and __writing proper tests__ are also highly welcome.
+
+### Contribution
+
+We appreciate any contribution, from fixing a grammar mistake in a comment to implementing complex algorithms. Please read this section if you are contributing your work.
+
+Your contribution will be tested by our [automated testing on Github CI/CD](https://github.com/SystemCraftsman/strimzi-kafka-cli/actions) to save time and mental energy.  After you have submitted your pull request, you should see the Github CI/CD tests start to run at the bottom of your submission page.  If those tests fail, then click on the ___details___ button and try to read through the Github CI/CD output to understand the failure.  If you do not understand, please leave a comment on your submission page and a community member will try to help.
+
+Please help us keep our issue list small by adding fixes: #{$ISSUE_NO} to the commit message of pull requests that resolve open issues. GitLab will use this tag to auto-close the issue when the PR is merged.
+
+# Getting Started with Development
+
+### Cloning the Repository
+
+To start your development journey, you will need to clone the repository from Github. Cloning a repository creates a local copy of the project on your machine, allowing you to make changes and contribute to the codebase.
+
+Follow these steps to clone the repository:
+
+1. Open your web browser and navigate to the repository on Github.
+2. On the repository page, click on the Fork button in the top-right corner. This will create a personal copy of the repository under your GitLab account.
+3. Once the repository is forked, go to your GitLab profile and navigate to the forked repository.
+4. Click on the "Clone" button for clone to repository.
+5. Copy the URL provided in the cloning options. It should look like https://github.com/your-username/strimzi-kafka-cli.git.
+6. Open your terminal or command prompt on your local machine.
+7. Navigate to the directory where you want to clone the repository using the cd command. For example, to navigate to your home directory, you can use cd ~.
+8. In the terminal, enter the following command to clone the repository:
+
+```bash
+git clone https://github.com/your-username/strimzi-kafka-cli.git
+```
+Replace `your-username` with your GitLab username.
+
+9. Press Enter to execute the command. Git will now download the repository and create a local copy on your machine.
+
+10. Once the cloning process is complete, you can navigate into the cloned repository using the cd command. For example:
+
+```bash
+cd strimzi-kafka-cli
+```
+
+Congratulations! You have successfully cloned the repository and are ready to start development. In the next sections, we will cover the setup and configuration steps required for your development environment.
+
+
+# Setting Up Poetry
```

Comment:
@onuralpszr If you can change the `poetry` relevant steps in this PR, we can merge this as well